library/src/interactor_impl.cxx

```diff
@@ -273,6 +273,18 @@ class interactor_impl::internals
       filesVec[i] = filesArr->GetValue(i);
     }
 
+    vtkRenderWindowInteractor* rwi = self->Style->GetInteractor();
+    if (rwi->GetControlKey() == 1)
+    {
+      // When Ctrl Key is pressed, load the first file as an HDRI instead
+      // of loading a file.
+      // This bypass the DropFilesUserCallBack logic and will be reworked
+      // once https://github.com/f3d-app/f3d/issues/443 is addressed.
+      self->Options.set("render.background.hdri", filesVec[0]);
+      self->Window.render();
```

Comment:
Can you add a test to cover this code?

Reply:
As stated in the MR description, the interaction system does not seems to handle modifier well. I hope to investigate that in VTK at some point.

Reply:
who read MR description? 🤷